src/vm/jithelpers.cpp

```diff
@@ -2845,6 +2845,69 @@ HCIMPLEND
 
 //========================================================================
 //
+//      ALLOCATION FAST PATHS
```

Comment:
I think this should be in under ALLOCATION HELPERS - to match how the rest of the file is structured.